api/debug/debug.go

```diff
@@ -215,6 +198,28 @@ func (d *Debug) handleTraceCall(w http.ResponseWriter, req *http.Request) error
 	return utils.WriteJSON(w, res)
 }
 
+type extractTracerOption struct {
+	Name   string
+	Config json.RawMessage
+}
+
+func extractTracer(opt *extractTracerOption, tracer *tracers.Tracer) error {
```

Comment:
How about `(d *Debug) createTracer(name string, config json.RawMessage)(tracers.Tracer, error)` ?

Reply:
Are there any advantages in making it a struct method? Because when I created it I thought about it as a utility function to extract the tracer from config, and not as a part of the debug struct, but I might be missing something

Reply:
One reason is keep the same style of all API packages, another one is sharing the option `d. allowCustomTracer `

Reply:
got it, done